WORKSPACE

```diff
@@ -82,14 +82,6 @@ container_pull(
     tag = "3.8",
 )
 
-container_pull(
-    name = "k8s_pause_amd64",
-    # this is a manifest list, so the resolved digest should not match this digest
-    digest = "sha256:f78411e19d84a252e53bff71a4407a5686c46983a2c2eeed83929b888179acea",
```

Comment:
lets try to leave this test but remove the digest attr?

Reply:
Removing the digest attr here will then use the default "latest" tag, which will be hard to test with `file_test`.
And we already have a test that relies on a tag (one target below called `k8s_pause_arm64`).
Let me know if you still want to keep this `container_pull` target without the digest attr and leave it untested.

Reply:
sgtm